source/mod_fluc.f90

```diff
@@ -86,12 +84,6 @@ subroutine fluc_parseInputLine(inLine, iLine, iErr)
   fluc_iSeed1 = izu0
   fluc_iSeed2 = 0
 
-  if(mmac > nmac) then
-    write(lout,"(a,i0)") "FLUC> ERROR Maximum number of seeds for vectorisation is ",nmac
-    iErr = .true.
-    return
-  end if
-
```

Comment:
shouldn't we anyway leave the error message if someone is crazy enough to ask more than `mmac=1`? At least, until the `SIMUL` block will be the driving one for simulation parameters...

Reply:
The value isn't even read. It's skipped. Also, the variable mmac is deleted.

Or did you mean in case someone thought it still existed and tried to use it? It is described in the manual.

Reply:
Added the error message back in. Made it for `mmac > 1` only, so it will accept 0 as well. Basically, if someone tries to make it deliberately more than 1, it will error.